src/modules/Dropdown/Dropdown.js

```diff
@@ -1047,7 +1047,9 @@ export default class Dropdown extends Component {
     } else if (hasValue) {
       _text = _.get(this.getItemByValue(value), 'text')
     }
-
+    if (classes === 'text') {
+      return <div className={classes} role='alert' aria-live='polite'>{_text}</div>
+    }
```

Comment:
I update the code. Now returns always the rendered text with the aria-labels suggested. I tested the documentation page, and all dropdown added work as in the main documentation page (http://react.semantic-ui.com)